foreman/Dockerfile.tests

```diff
@@ -1,6 +1,6 @@
 # This Dockerfile is meant to be run from the directory above where it resides.
 
-FROM python:3.5
+FROM python:3.6.1-slim
```

Comment:
Hmm good call, I made some fairly large changes based on that feedback and am very happy with the way it turned out.